.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,7 +27,7 @@ What kind of change does this PR introduce?
 Please check if your PR fulfills the following requirements:
 
 - [ ] Tested code with current [supported SDKs](../readme.md#supported)
-- [ ] Pull Request has been submitted to the documentation repository [instructions](..\contributing.md#docs). Link: <!-- docs PR link -->
+- [ ] Pull Request has been submitted to the documentation repository [instructions](../blob/master/contributing.md#-adding-documentation). Link: <!-- docs PR link -->
```

Comment:
The other ones are also not working. We should add /blob/master/ to all of them ("supported SDKs" and "Thumbnail Style Guide and templates" links too).

Reply:
only the SDKs link was broken. The template style link works